SwiftWisdom/Core/Foundation/Data/Data+Extensions.swift

```diff
@@ -16,12 +16,13 @@ extension Data {
         guard self.count > 0 else { return nil }
         return map { String(format: "%02hhx", $0) }.joined()
     }
+
+    // MARK: Convenience
     
     // From here: http://stackoverflow.com/questions/38023838/round-trip-swift-number-types-to-from-data
```

Comment:
Yes, but it is already in the DataConvertible protocol where it belongs, so we can remove it from here.